### DISCUSSION TITLE
Excentric visual for unordered list and <details>

### DISCUSSION THREAD

**Discussion by despens** [requarks/wiki#3643] [Error / Bug Report]
**Describe the bug**

All unordered lists use a caret character `▸` for the "bullet" graphic. This makes list items look as if they are expandable.

On the other hand, the actually expandable `<details>` feature, which browsers by default display with a caret, doesn't give any visual hint that it is expandable in Wikijs.

**To Reproduce**
Create a page with the following markdown:

```markdown
- Learn more

<details>
<summary>The details you would need to know</summary>
  
- details
  - more details
  - extreme details
  
</details>
```

**Expected behavior**
I would expect the following rendering:

- Learn more

<details>
<summary>The details you would need to know</summary>
  
- details
  - more details
  - extreme details
  
</details>

**Screenshots**

GitHub rendering:
![image](https://user-images.githubusercontent.com/571494/76525089-81316e00-646b-11ea-8401-04bbcae0b630.png)

Wikijs rendering:
![image](https://user-images.githubusercontent.com/571494/76525133-960e0180-646b-11ea-98dd-5bc68bb39e1f.png)

**Additional context**
Standard rendering of these elements would probably be best to avoid confusion.

**Comment by dserodio** [comment]
I agree 100%. I didn't find this post before so I asked https://github.com/requarks/wiki/discussions/5745 where I managed to get an answer for how to customize the CSS to get unordered list items to look like unordered list items, but the default should be sane.

IMHO, the current formatting of unordered list items is a bug.

I didn't even try to create a details/summary because I figured it would be "disguised" as an unordered list item, but know that I see your screenshot, it doesn't look like a collapsable summary/details at all.

I just tried in on my local install, and it's rendering differently from your screenshot: a little better, but still **very** confusing 🤔 

![image](https://user-images.githubusercontent.com/233589/219113331-82e1cd1e-9897-4a9f-9c5b-d69b1894bcd4.png)
